packages/kbn-test/src/functional_tests/tasks.js

```diff
@@ -0,0 +1,140 @@
+import { relative, resolve } from 'path';
+import Rx from 'rxjs/Rx';
+import getopts from 'getopts';
+import { withProcRunner } from '@kbn/dev-utils';
+
+import {
+  withTmpDir,
+  runKibanaServer,
+  runElasticsearch,
+  runFtr,
+  log,
+  KIBANA_ROOT,
+  KIBANA_FTR_SCRIPT,
+} from './lib';
+
+import { readConfigFile } from '../../../../src/functional_test_runner/lib';
+
+const SUCCESS_MESSAGE = `
+
+Elasticsearch and Kibana are ready for functional testing. Start the functional tests
+in another terminal session by running this command from this directory:
+
+    node ${relative(process.cwd(), KIBANA_FTR_SCRIPT)}
+
+`;
+
+
+/**
+ * Run servers and tests for each config
+ * @param {configPaths}     array of paths to configs
+ * @param {options}         optional
+ * @param {options.runEs}   method to start elasticsearch
+ * @param {options.runKbn}  method to start kibana
+ * }
+ */
+export async function runTests(
+  configPaths,
+  options = {
```

Comment:
__issue:__ Hmm, something feels wrong here, how would you override __only__ `runEs` with this signature? Shouldn't the signature be like this instead?

```js
{ runEs = runElasticsearch, runKbn = runKibanaServer } = {}
```

Reply:
Honestly I'm considering removing this entirely. The use case for injecting runEs/runKbn has gone away, and I don't see a reason to design for a future to allow it. We can do that when we need it.

Reply:
What do you think? Just remove the options arg entirely, @spalger @azasypkin? unless someone really likes it...

Reply:
Nevermind. we still need runEs for xpack only.

Reply:
> Honestly I'm considering removing this entirely. The use case for injecting runEs/runKbn has gone away, and I don't see a reason to design for a future to allow it. We can do that when we need it.

If we don't need these args, that's even better!